packages/lambda-xyz/src/routes/tile.ts

```diff
@@ -107,7 +107,8 @@ export async function Tile(req: LambdaContext, xyzData: TileDataXyz): Promise<La
 
     if (!Env.isProduction()) {
```

Comment:
Would it be nicer here to either just test if debug is turned on or test both?

Reply:
Eventually i think we should possibly have a `DEBUG=:area` and let that configure the debugging logging for specific areas of the code base.

eg `DEBUG=tiler` or something